etc/DependencyInstaller.sh

```diff
@@ -274,7 +411,7 @@ _installOrTools() {
 
     # Disable exit on error for 'find' command, as it might return non zero
     set +euo pipefail
-    LIST=($(find / -type f -name "libortools.so*" 2>/dev/null))
+    _searchEverywhere --update-db --file-pattern "libortools.so*"
```

Comment:
--update-db will trigger `sudo updatedb` even if locate isn't installed.  This should not be required in that case.